app/views/travel_advice/index.html.erb

```diff
@@ -27,16 +27,14 @@
             </div>
             <div class="travel-advice-notice__content">
               <p class="govuk-body">
-                Follow current COVID-19 rules where you live: <a href="/guidance/covid-19-coronavirus-restrictions-what-you-can-and-cannot-do" class="govuk-link">England</a>, <a href="https://www.gov.scot/coronavirus-covid-19/" class="govuk-link">Scotland</a>, <a href="https://gov.wales/coronavirus-travel" class="govuk-link">Wales</a> and <a href="https://www.nidirect.gov.uk/articles/coronavirus-covid-19-regulations-guidance-what-restrictions-mean-you" class="govuk-link">Northern Ireland</a>.
+                To understand the risks in a country, including the latest COVID restrictions (including for entry), follow FCDO Travel Advice.
```

Comment:
I notice on the design this has a link to this. It seems the right decision to not link here but I wonder if this is going to surprise the people who put the copy together as they might not have realised this quirk.

Reply:
We've spotted this before, and always remove the link on the search page as it links to itself and is a worse user experience.

Reply:
Great - glad there's precedence.